diff --git a/node_modules/react-native-reanimated-carousel/lib/typescript/layouts/parallax.d.ts b/node_modules/react-native-reanimated-carousel/lib/typescript/layouts/parallax.d.ts
index f04c70f..3bd6ff5 100644
--- a/node_modules/react-native-reanimated-carousel/lib/typescript/layouts/parallax.d.ts
+++ b/node_modules/react-native-reanimated-carousel/lib/typescript/layouts/parallax.d.ts
@@ -19,6 +19,8 @@ export interface ILayoutConfig {
        * @default Math.pow(parallaxScrollingScale, 2)
        */
     parallaxAdjacentItemScale?: number;
+    
+    parallaxScrollingInverted?: boolean;
 }
 export declare type TParallaxModeProps = IComputedDirectionTypes<{
     /**
diff --git a/node_modules/react-native-reanimated-carousel/src/layouts/ParallaxLayout.tsx b/node_modules/react-native-reanimated-carousel/src/layouts/ParallaxLayout.tsx
index 3eadaab..259b3ba 100644
--- a/node_modules/react-native-reanimated-carousel/src/layouts/ParallaxLayout.tsx
+++ b/node_modules/react-native-reanimated-carousel/src/layouts/ParallaxLayout.tsx
@@ -30,6 +30,7 @@ IComputedDirectionTypes<
     parallaxScrollingOffset = 100,
     parallaxScrollingScale = 0.8,
     parallaxAdjacentItemScale = parallaxScrollingScale ** 2,
+    parallaxScrollingInverted = false,
     index,
     width,
     height,
@@ -94,7 +95,7 @@ IComputedDirectionTypes<
             translateY: translate,
           }
           : {
-            translateX: translate,
+            translateX: parallaxScrollingInverted ? -translate : translate,
           },
         {
           scale,
diff --git a/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts b/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts
index 2f8246f..ea8c5c1 100644
--- a/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts
+++ b/node_modules/react-native-reanimated-carousel/src/layouts/parallax.ts
@@ -23,6 +23,8 @@ export interface ILayoutConfig {
      * @default Math.pow(parallaxScrollingScale, 2)
      */
   parallaxAdjacentItemScale?: number
+
+  parallaxScrollingInverted?: boolean
 }
 
 export type TParallaxModeProps = IComputedDirectionTypes<{
@@ -42,6 +44,7 @@ export function parallaxLayout(
     parallaxScrollingOffset = 100,
     parallaxScrollingScale = 0.8,
     parallaxAdjacentItemScale = parallaxScrollingScale ** 2,
+    parallaxScrollingInverted = false,
   } = modeConfig;
 
   return (value: number) => {
@@ -77,7 +80,7 @@ export function parallaxLayout(
             translateY: translate,
           }
           : {
-            translateX: translate,
+            translateX: parallaxScrollingInverted ? -translate : translate,
           },
         {
           scale,
